akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -285,6 +285,11 @@ object Flow {
   def apply[T]: Flow[T, T, NotUsed] = identity.asInstanceOf[Flow[T, T, NotUsed]]
 
   /**
+   * Creates a `Flow` from a function.
```

Comment:
`Creates a `Flow` which will use the given function `f` to transform its inputs to outputs.`